Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,11 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderTitle = props =>
+{
+    return (
+        <h3>Lambda School</h3>
+    );
```

Comment:
This is a good start to the social card. I'd like to see the rest of the content added to finish it out.